x-pack/plugins/security_solution/server/lib/telemetry/filterlists/endpoint_alerts.ts

```diff
@@ -18,6 +18,8 @@ const baseAllowlistFields: AllowlistFields = {
   env_vars: true,
   hash: true,
   pid: true,
+  origin_referrer_url: true,
```

Comment:
Shouldn't it be located under `file` in `allowlistBaseEventFields`?

We want to filter for instance `events.file.origin_referrer_url`, `Events.file.origin_referrer_url`, or `file.origin_referrer_url`, right?

`baseAllowlistFields` configures `process`, `process.parent`, `Target.process` and `Target.process.parent`  🤔

Reply:
@szaffarano 
Thank you for the review! 
As you pointed out, the location where I added the field was incorrect. Following your suggestion, I have added the field to allowlistBaseEventFields.